lib/cstring.h

```diff
@@ -254,17 +254,11 @@ class cstring {
     }
     cstring substr(size_t start, size_t length) const;
     cstring replace(char find, char replace) const;
-    cstring replace(cstring find, cstring replace) const;
+    cstring replace(std::string_view find, std::string_view replace) const;
```

Comment:
This might be the only breaking change in this PR, right?

Reply:
Yes. Though if before explicit-cstring-construction it was like `replace("foo", "bar")`, then after this PR it will start working again w/o changes :)

Reply:
Hmm, I wonder why not return a `std::string` here. In many cases, we would not need the result to be `cstring` and if the caller needs it, they can always do that explicitly.

Reply:
I checked this case, and there is a mixture of usages. So I decided to re-evaluate this when we'll disable implicit conversion to / from std::string – then I will catch all such places :)